### PR TITLE
Add feature screenshots, demo video, and slide functionality

### LINKED ISSUE #137: Add screenshots + video

### PR DESCRIPTION
**Pull request by SeanLeRoy**
**Description**
This changeset adds the screenshots to the splash page for the feature carousel. This is intended be used as a way to quickly introduce people to the capabilities of the application. In addition to the actual screenshots I added functionality to maneuver between slides of features.

**Testing**
Tested locally, [see here](http://10.128.53.95:8080/)

**Related Issue**
Resolves #137

### CODE REVIEW

File: packages/web/src/components/Home/index.tsx [hunk 1]
```diff
@@ -57,10 +57,13 @@ export default function Home() {
                         </div>
                     </div>
                     <div className={styles.videoWalkthrough}>
-                        <div className={styles.videoWalkthroughFake}>
-                            Video placeholder: Demo showing how to use the app via interactive
-                            walkthrough
-                        </div>
+                        <iframe
+                            src="https://player.vimeo.com/video/974452570?badge=0&amp;autopause=0&amp;player_id=0&amp;app_id=58479"
```
Comment (aswallace) [inline packages/web/src/components/Home/index.tsx:61]: Current link requires login to Vimeo. Is there a way to make it public?

Comment (SeanLeRoy) [reply packages/web/src/components/Home/index.tsx:61]: Try now, I changed the permissions

File: packages/web/src/components/Home/FeatureOptions.ts [hunk 2]
```diff
@@ -1,37 +1,119 @@
...
     },
     {
         id: 4,
         text: "Share via URL, code snippet, or download",
-        caption: "Placeholder text about sharing code snippets",
+        slides: [
+            {
+                imgSrc: "/Share/URL.png",
+                caption:
+                    "Your exact view (filters, groups, sorts, open folders) can be shared via URL. This URL can be shared with anyone and they will see the exact same view you are seeing assuming the data source is accessible to them. Local files for example will have to be re-selected by the user.",
```
Comment (aswallace) [inline packages/web/src/components/Home/FeatureOptions.ts:89]:
```suggestion
                    "Your exact file selection (filters, groups, sorts, open folders) can be shared via URL. Anyone with the URL can see the same view as you, as long as the data source is accessible to them. Local files, for example, must be re-selected by the user.",
```

File: packages/web/src/components/Home/FeatureOptions.ts [hunk 3]
```diff
@@ -1,37 +1,119 @@
...
+            },
+            {
+                imgSrc: "/Share/CodeSnippet.png",
+                caption:
+                    "A code snippet can be generated that will recreate your exact view (filters, groups, sorts) programmatically. This could be useful for sharing your view with a colleague who is working in a Jupyter notebook for example.",
```
Comment (aswallace) [inline packages/web/src/components/Home/FeatureOptions.ts:94]:
```suggestion
                    "A code snippet can be generated that recreates your exact view (filters, groups, sorts) programmatically. This allows you to export or share your view to a Jupyter notebook.",
```

File: packages/web/src/components/Home/FeatureOptions.ts [hunk 4]
```diff
@@ -1,37 +1,119 @@
...
+            },
+            {
+                imgSrc: "/Share/Download.png",
+                caption:
+                    "Lastly, BioFile Finder can be used as an interface for downloading files. Select the file you want to download, right-click to open the context menu, and select 'Download' to download the focused file.",
```
Comment (aswallace) [inline packages/web/src/components/Home/FeatureOptions.ts:99]:
```suggestion
                    "You can also download files directly from BioFile Finder. Select the file(s) you want to download, right-click to open the context menu, and select 'Download.'",
```

File: packages/web/src/components/Home/FeatureOptions.ts [hunk 5]
```diff
@@ -1,37 +1,119 @@
...
+            },
+        ],
     },
     {
         id: 5,
-        text: "View OME.Zarr files instantly using the web-based 3D Volume Viewer",
-        caption: "Placeholder text about viewing OME.Zarr files",
+        text: "View OME.Zarr or pre-generated thumbnails previews of files instantly",
```
Comment (aswallace) [inline packages/web/src/components/Home/FeatureOptions.ts:105]:
```suggestion
        text: "View OME.Zarr or pre-generated thumbnail previews of files instantly",
```

File: packages/web/src/components/Home/index.tsx [hunk 6]
```diff
@@ -57,10 +57,13 @@ export default function Home() {
...
+                            width="267"
```
Comment (aswallace) [inline packages/web/src/components/Home/index.tsx:62]: This seems a little narrower than the mockup, though might just be on my screen

File: packages/web/src/components/Home/FeatureOptions.ts [hunk 7]
```diff
@@ -1,37 +1,119 @@
...
+            },
+            {
+                imgSrc: "/Datasets/AddDataSource.png",
+                caption:
+                    "Adding a second, third, fourth, etc. data source works by simply repeating the process.",
+            },
+            {
+                imgSrc: "/Datasets/GroupedDataSources.png",
+                caption:
+                    'After you have added your data you can filter, group, and sort it however; this screenshot shows grouping by an automatically generated column "Data source" which represents the Data source each file came from.',
```
Comment (aswallace) [inline packages/web/src/components/Home/FeatureOptions.ts:52]:
```suggestion
                    'After you have added data, you can filter, group, and sort it. This screenshot shows an example of grouping by an automatically generated column, "Data source," which represents the source of each file.',
```

**Review by aswallace — COMMENTED** [review]
Left some suggested edits! Mostly nits

File: packages/web/src/components/Home/index.tsx (hunk 6 above)
Comment (SeanLeRoy) [reply packages/web/src/components/Home/index.tsx:62]: Yeah - it couldn't be the same width and height so I chose the same height lmk if you think I should've done width instead (aspect ratio is why)

Comment (aswallace) [reply packages/web/src/components/Home/index.tsx:62]: Makes sense! I'd vote for whichever one makes it bigger overall

File: packages/web/src/components/Home/FeatureOptions.ts [hunk 8]
```diff
@@ -1,37 +1,119 @@
...
+                imgSrc: "/FindingData/Group.png",
+                caption: `Unlike traditional file systems BioFile Finder allows you to dynamically generate your folder structure based on the metadata of your files. In this screenshot, the files are organized by (Grouped by) the "Gene" and "Structure" columns found in the selected Data source.`,
+            },
+            {
+                imgSrc: "/FindingData/FilterColumn.png",
+                caption: `Filter by selecting a column to filter by...`,
+            },
+            {
+                imgSrc: "/FindingData/FilterValue.png",
+                caption: `...then selecting a value for that column to filter by.`,
```
[the remark this answers is not in the document]
Comment (SeanLeRoy) [reply packages/web/src/components/Home/FeatureOptions.ts:27]: I like it 👍

File: packages/web/src/components/Home/FeatureOptions.ts [hunk 9]
```diff
@@ -1,37 +1,119 @@
...
     {
         id: 2,
-        text: "Create new or combine subsets",
-        caption: "Placeholder text about creating or combining",
+        text: "Create new data sources or combine them",
+        slides: [
+            {
+                imgSrc: "/Datasets/LoadSource.png",
+                caption:
+                    "Data can be loaded from a CSV, Parquet, or JSON file. The data source must include a column that contains the file path to the data. The data source can be loaded from a URL or uploaded from your local machine.",
```
[the remark this answers is not in the document]
Comment (SeanLeRoy) [reply packages/web/src/components/Home/FeatureOptions.ts:42]: Adjusted 👍

**Review by aswallace — APPROVED** [review]

File: packages/web/src/components/Home/FeatureOptions.ts [hunk 10]
```diff
@@ -1,37 +1,118 @@
...
         id: 1,
         text: "Find target files quickly",
-        caption: `Filter, sort, group or search for your files
-      in a predictable and standard way and view thumbnails for 
-      quick identification. Switch to thumbnail view for...this 
-      is placeholder text for now.`,
+        slides: [
+            {
+                imgSrc: "/FindingData/Group.png",
+                caption: `Unlike traditional file systems, BioFile Finder allows you to dynamically generate your folder structure based on the metadata of your files. In this screenshot, the files are organized by (Grouped by) the "Gene" and "Structure" columns found in the selected data source.`,
```
Comment (BrianWhitneyAI) [inline packages/web/src/components/Home/FeatureOptions.ts:19]: I don't think we need to start with "Unlike traditional file systems," I would just start with BioFile Finder allows...

**Review by BrianWhitneyAI — APPROVED** [review]
LGTM! A couple rewording things but most of them are just prefrence.

**Pull request merged**